tests/test_applicationbuilder.py

```diff
@@ -305,7 +306,7 @@ def test_all_bot_args_custom(self, builder, bot, monkeypatch):
         # other means that the parameters are passed correctly
 
         assert built_bot.token == bot.token
-        assert built_bot.base_url == "base_url" + bot.token
+        assert docs.auxil.link_code.base_url == "base_url" + bot.token
```

Comment:
why not just use `built_bot`? how is this different?

Reply:
Irg, probably a search&replace mistake ...